mayavi/modules/volume.py

```diff
@@ -455,10 +455,15 @@ def _setup_current_range(self):
         # Set the current range.
         dataset = mm.source.get_output_dataset()
         dsh = DataSetHelper(dataset)
-        name, rng = dsh.get_range('scalars', 'point')
-        if name is None:
+        point_name, point_rng = dsh.get_range('scalars', 'point')
+        cell_name, cell_rng = dsh.get_range('scalars', 'cell')
+        if point_name is None and cell_name is None:
             error('No scalars in input data!')
             rng = (0, 255)
+        elif not point_name is None:
```

Comment:
This should be `elif point_name is not None:`

Reply:
Fixed.